app/assets/stylesheets/components/tabs.scss

```diff
@@ -70,7 +70,7 @@
     }
 
     &--current {
-      font-weight: var(--fw-medium);
+      text-shadow: 0 0 0.075em currentColor;
```

Comment:
This is a great idea - tiny layout shifts make my eye twitch 😄 

Only thing I would suggest, is currently the third value is specifying a blur effect, and it looks a bit fuzzy to me. I think if we change it a little to have 0 blur then it looks a bit sharper and closer to the original look:

```suggestion
      text-shadow: 0 0.075em currentColor;
```

Screenshot before:

<img width="1300" alt="screenshot showing the admin analytics tabs with slightly fuzzy bold text" src="https://user-images.githubusercontent.com/20773163/113892407-dac6b200-97bd-11eb-9418-8cd90a41c2e9.png">

Screenshot after:

<img width="1302" alt="screenshot showing the suggested alternative with slightly less of a fuzzy look" src="https://user-images.githubusercontent.com/20773163/113892426-def2cf80-97bd-11eb-8c41-ff49a36648da.png">

At small UI sizes it's not quite so visible, but on larger screens it makes quite a big difference

Reply:
I changed it to text stroke to solve that same issue 😅

Looks like it's supported every but in IE.

Reply:
I'm back at a computer now so I can explain my logic `text-shadow` takes x offset, y offset and blur none of which are ideal, though you're right using the y offset looks better than using the blur. To start with I was a little confused as to why `text-shadow` didn't have a spread option like `box-shadow` then it occurred to me it didn't need one as it already had `text-stroke` so I used that.

src: [caniuse](https://caniuse.com/mdn-css_properties_-webkit-text-stroke)

Reply:
Had a quick look into `text-stroke` and it seems even though it's currently supported by modern browsers, it's still [not considered a 'standard' feature and not on a path to becoming one](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-stroke) - in which case I'm tempted to say we'd be better off running with the `text-shadow` approach as it seems more future-proof. What do you think?

Reply:
What do you think to doing something like this?

```scss
&--current {
  text-shadow: -0.009em -0.009em currentColor, 0.009em -0.009em currentColor, -0.009em 0.009em currentColor, 0.009em 0.009em currentColor;
}
```

I know the it's a little long winded but I think it looks more like the bold style it's trying to copy.

![tab example](https://user-images.githubusercontent.com/3534427/114386186-cea96e80-9b88-11eb-973f-1aeb49bc495b.png)

Reply:
Yeah I think that looks better! Maybe the `0.009em` could be brought out into a variable to make it more readable? not a strong opinion though - if we start needing this CSS elsewhere (I'm sure we will), we could just bring out into a variable then

Reply:
Good idea, I've added it to the normal variables now.

Reply:
Hey @Link2Twenty - I've been taking a look and it looks great in Chrome, but both Safari and Firefox look fuzzy:

This is in Safari:

![screenshot of the home feed tabs, 'week' is selected but looks a bit fuzzy](https://user-images.githubusercontent.com/20773163/116370123-fe4fac00-a801-11eb-9cf9-92148a712108.png)


Current Live site for ref:

![screenshot of the home feed tabs, with 'week' selected, looking bold but sharper than the last image](https://user-images.githubusercontent.com/20773163/116370097-f5f77100-a801-11eb-884f-f485b26b0b25.png)

Do you think there's a way round this? As much as the layout shift isn't great, I think we want to avoid decreasing the legibility of the tabs in the various browsers.

Reply:
I'm not sure there's any other way of getting around the bold layout shift. Unless we go the fixed width route?

Reply:
I had a play around and it seems each browser handles the values quite differently and there's not an ideal that looks good in all 🙁 

The only other suggestion I've been able to find is to use a hidden pseudo element which already has the bold text: https://css-tricks.com/bold-on-hover-without-the-layout-shift/

Maybe worth a try?

Reply:
Interesting it would mean we'd have to add a data attribute to every tab that contained the tab text again. Which isn't unreasonable I suppose.

Reply:
Yeah I think that could be doable - if someone implemented tabs without the data attribute, the worst that would happen is we'd have the same layout shift as now.

In saying that, I haven't tried this approach at all yet 😅

Reply:
Looks pretty good to me! I think we'd just need to add a comment in the CSS to explain why we're doing it, but it seems to work well 👍

Reply:
Found a couple of minute to implement if you wanna have a look.